cluster-scope/base/namespaces/opf-ci-prow/namespace.yaml

```diff
@@ -4,7 +4,17 @@ kind: Namespace
 metadata:
   annotations:
     openshift.io/display-name: "opf-ci-prow"
-    openshift.io/discription: "Project for execution of continous integration prow jobs."
+    openshift.io/discription: "Project for Prow and the gang."
     openshift.io/requester: aicoe-thoth-devops
   name: opf-ci-prow
 spec: {}
+---
```

Comment:
Having multiple resources in a single file won't apply the project admin component from the `kustomization.yaml` properly, you can't have it here. The way you've done it at https://github.com/operate-first/apps/pull/221 PR was much better.